src/main/java/org/folio/auth/authtokenmodule/UserService.java

```diff
@@ -28,7 +29,10 @@ public UserService(Vertx vertx, int cacheInSeconds, int purgeCacheInSeconds) {
     cachePeriod = cacheInSeconds * 1000L;
 
     // purge cache periodically
-    vertx.setPeriodic(purgeCacheInSeconds * 1000L, id -> cache = new ConcurrentHashMap<>());
+    vertx.setPeriodic(purgeCacheInSeconds * 1000L, id -> {
+      cache = new ConcurrentHashMap<>();
```

Comment:
Maybe rename this variable `activeUserCache` since it is no longer the only cache.

Reply:
Renamed, @steveellis can you please review this PR also: https://github.com/folio-org/mod-users-bl/pull/161 
It is very important for us, because user in institutional tenant could not setup password and so could not login using central tenant UI, we already discussed  this approach, there is no any security violations in exposting tenant to reset password link

Reply:
You're all set @SerhiiNosko. Approved the mod-users-bl one too.

Reply:
thank you